src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,36 @@
+package duke.task;
+
+public class Todo extends Task {
+    protected boolean isDone;
+    public Todo(String description) {
+        super(description);
+        isDone = false;
+    }
+    public void setDone(boolean done) {
+        isDone = done;
+    }
+    public boolean isDone() {
+        return isDone;
+    }
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
```

Comment:
The comments could be placed on a separate line with same indentation. Refer to the[ Java coding standard ](https://se-education.org/guides/conventions/java/basic.html#comments)for examples.